src/test/regress/sql/seclabel.sql

```diff
@@ -62,11 +62,20 @@ SET citus.grep_remote_commands = '%SECURITY LABEL%';
 SECURITY LABEL for "citus '!tests_label_provider" ON ROLE user1 IS 'citus_classified';
 SECURITY LABEL ON ROLE user1 IS NULL;
 SECURITY LABEL ON ROLE user1 IS 'citus_unclassified';
-SECURITY LABEL for "citus '!tests_label_provider" ON ROLE "user 2" IS 'citus ''!unclassified';
+SECURITY LABEL for "citus '!tests_label_provider" ON ROLE "user 2" IS 'citus_classified';
```

Comment:
I couldn't quite understand why we had to change the security label specified in an existing test case

Reply:
We did not have to, it just made easier to update the expected results. The new tests change labels of both users. That security label is applied from the worker in the following lines.